### PR TITLE
Better simulation of GET requests whenusing http method override

### PR DESCRIPTION
**Pull request by lorenzo**

### CODE REVIEW

File: src/Network/Request.php [hunk 1]
```diff
@@ -283,12 +285,20 @@ protected function _processPost($data)
...
         $this->_environment['ORIGINAL_REQUEST_METHOD'] = $method;
         if (isset($data['_method'])) {
             $this->_environment['REQUEST_METHOD'] = $data['_method'];
             unset($data['_method']);
+            $override = true;
+        }
+
+        $method = $this->_environment['REQUEST_METHOD'];
+        if ($override && !in_array($method, ['PUT', 'POST', 'DELETE', 'PATCH'])) {
+            $data = [];
```
Comment (markstory) [inline src/Network/Request.php]: What about input decoded from `php://input`? That will still have the same issue.

**Comment by ADmad** [comment]
Tests failing.

File: src/Network/Request.php (hunk 1 above)
Comment (lorenzo) [inline src/Network/Request.php]: input was already converted at this point. when it is urlencoded

**Comment by ADmad** [comment]
:+1:

**Pull request merged**